modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -5415,7 +5443,8 @@ private void _validateValues(
 			Map<String, Serializable> values)
 		throws PortalException {
 
-		if (Validator.isNull(value) && objectField.isRequired() &&
+		if (Validator.isNull(value) && !objectField.isLocalized() &&
+			objectField.isRequired() &&
```

Comment:
Why do we only want to throw this exception when it is not localized?
Shouldn't we validate when it is localized, too?

Reply:
We are already doing the required validation for the localized fields in `_insertIntoLocalizationTable` (in the comment above). We are also throwing a different exception for localized fields: `RequiredLanguageId`. 
And we do not have all the information we need to validate in this other point. So, I thought it was best to do the validation in one place.